src/config/exampleConfig.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Why did you remove the example config...

Reply:
yeah i changed it to config, will revert it back